src/components/notification/NotificationsWrapper.js

```diff
@@ -1,12 +1,19 @@
 // @flow
 import * as React from 'react';
 
+import FocusTrap from '../focus-trap';
 import Portal from '../portal';
 
 type Props = {
     children?: React.Node,
 };
 
-const NotificationsWrapper = ({ children }: Props) => <Portal className="notifications-wrapper">{children}</Portal>;
+const NotificationsWrapper = ({ children }: Props) => (
+    <FocusTrap>
```

Comment:
I'm not super familiar with these components. Does the FocusTrap render around the correct part of the DOM, given that its children are rendered in a portal? Or should it be more like:

```
 <Portal className="notifications-wrapper" aria-live="polite">
    <FocusTrap>
        {children}
    </FocusTrap>
</Portal>
```

Reply:
Yes , that is correct. I just moved it inside. You just saved me some extra debugging time.